prog/phonons/initphonons.F90

```diff
@@ -145,25 +145,25 @@ module dftbp_initphonons
   integer, parameter :: parserVersion = 4
 
   !> Version of the oldest parser, for which compatibility is maintained
-  integer, parameter :: minVersion = 4 
+  integer, parameter :: minVersion = 4
 
   !> Container type for parser related flags.
   type TParserFlags
```

Comment:
Might be worth converting comments in the TParserFlags type to doxygen/ford style.